build-plugin/src/main/kotlin/AndroidExtension.kt

```diff
@@ -0,0 +1,29 @@
+import com.android.build.api.dsl.CommonExtension
+
+fun CommonExtension<*, *, *, *>.configureSharedConfig() {
+    compileSdk = ThunderbirdProjectConfig.androidSdkCompile
+
+    defaultConfig {
+        compileSdk = ThunderbirdProjectConfig.androidSdkCompile
+        minSdk = ThunderbirdProjectConfig.androidSdkMin
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables.useSupportLibrary = true
+    }
+
+    compileOptions {
+        sourceCompatibility = ThunderbirdProjectConfig.javaVersion
+        targetCompatibility = ThunderbirdProjectConfig.javaVersion
+    }
+
+    lint {
+        checkDependencies = true
```

Comment:
This should probably only be set on an app module. Otherwise every module runs lint on itself and all its module dependencies.